v-next/hardhat-viem/src/accounts.ts

```diff
@@ -0,0 +1,22 @@
+import type { EthereumProvider } from "@ignored/hardhat-vnext/types/providers";
+import type * as viemT from "viem";
```

Comment:
Could we import `Address` directly here?

Reply:
I’ve done it this way to make it easier to differentiate when a type comes from `viem` versus `hardhat-viem`, since all viem types follow the format `viemT.Type`. I don’t believe this impacts performance, as it’s just type annotations, but I think this is the only plugin that uses this approach. I don’t have a strong opinion on keeping it, so I’ll leave the decision to you. If we decide to change it, I’d suggest being consistent and applying it across other files. wdyt?

Reply:
I think this is a nitpick and finely balanced.

My instinct here is to be brutally verbose with something like:

```ts
import type { Address as ViemAddress } from "viem";
```

And I come down on this side, and only just, because it gives us a general rule so that you can treat `import type * as xxx` as suspicious, rather than almost always suspicious but not in these case X,Y,Viem.

I am happy either way on this one - make a call and then feel free to resolve this conversation.

Reply:
Done [here](https://github.com/NomicFoundation/hardhat/pull/5825/commits/45cddb988679b153bc4edadd4af083d85e46a151). We should apply the same changes to other plugins, like `hardhat-ethers`.